docs/introduction/stability-support.md

```diff
@@ -50,6 +50,7 @@ The following table describes the stability level of each provider and who's res
 | [Oracle Vault](https://external-secrets.io/latest/provider/oracle-vault)                                   |   alpha   |                                                                                                 [@KianTigger](https://github.com/KianTigger) [@EladGabay](https://github.com/EladGabay) |
 | [Akeyless](https://external-secrets.io/latest/provider/akeyless)                                           |  stable   |                                                                                                                                 [external-secrets](https://github.com/external-secrets) |
 | [1Password](https://external-secrets.io/latest/provider/1password-automation)                              |   alpha   |                                                                                       [@SimSpaceCorp](https://github.com/Simspace) [@snarlysodboxer](https://github.com/snarlysodboxer) |
```

Comment:
Should we deprecate the connect token method? 🤔 It is clearly worse in everyu sense vs the SDK one.

@SimSpaceCorp @snarlysodboxer FYI - do you have thoughts regarding this?

Reply:
(I guess only when we have a full implemented featureset...)

Reply:
The SDK doesn't support deleting and pushing... So sadly no. 🤔

Reply:
Ah you literally said that on your second comment. Sorry.

Reply:
Oaky, so I had a previous comment about pushing secrets and deleting secrets.

This is NOT supported by the SDK. Items management is... But Items != Secrets.

Reply:
Ah. This is super weird. It IS supported. But will require some massaging. https://developer.1password.com/docs/sdks/manage-items

Might even have to do some json building or something I don't know. Because a secret may consist of many fields. Anyways, I'll see to it. :)

Reply:
Okay, adding these, just writing tests then this should be done. :)